app/views/posts/index.html.erb

```diff
@@ -1,2 +1,27 @@
-<h1>Posts#index</h1>
-<p>Find me in app/views/posts/index.html.erb</p>
+<div id="posts_container">
+  <div id="user_container">
+  <img id="user_img" src="<%= @user.photo %>" alt="user profile">
+      <div id="container_container_div">
+        <h3><%= @user.name %></h3>
+        <p>Number of posts: <%= @user.posts_counter %></p>
+      </div>
+    </div>
+    <% @user.posts.each do |post|%>
+    <div id="user_posts">
+      <h3><%= link_to post.class, user_post_path(@user.id, post.id)%><%= post.id %></h3>
+      <p><%= post.text %></p>
+      <h4>Comments <%= post.comments_counter %> likes: <%= post.likes_counter %></h4>
+    </div>
+    <% @comments=Comment.where(post_id: post.id)%>
```

Comment:
## OPTIONAL
* I strongly suggest you fix this bug I get when I try to add comments in the console below is a screenshot 👇 
![image](https://user-images.githubusercontent.com/52497246/216555572-d13edf5d-3958-4faf-8d8f-602116c09f43.png)